tests/reftests/switch-set.test

```diff
@@ -70,10 +70,8 @@ Switch ${BASEDIR} and all its packages will be wiped. Are you sure? [Y/n] y
 ### opam switch remove foo --yes
 Switch foo and all its packages will be wiped. Are you sure? [Y/n] y
 ### opam switch -
-[ERROR] The previous switch 'foo' no longer exists
-# Return code 5 #
+Selecting opam switch -.
 ### opam switch - --show
-[ERROR] The previous switch 'foo' no longer exists
-# Return code 5 #
+Would select opam switch bar.
```

Comment:
I’m not a big fan of the new behaviour but given it’s only going to be used very rarely, I’m fine with it.